transcription/transcription_test.go

```diff
@@ -0,0 +1,55 @@
+package transcription
+
+import (
+	"errors"
+	"net/smtp" // mock
```

Comment:
This is how `withmock` works. It scans the imports for the `// mock` comment to determine which packages to mock. Do you think I should document this better?